lib/modules/platform/bitbucket-server/index.ts

```diff
@@ -153,6 +158,41 @@ export async function getJsonFile(
   return JSON.parse(raw);
 }
 
+function getRepoGitUrl(
```

Comment:
Can you move this to the `./utils.ts` file please

Reply:
Hi @viceice ,
I have been trying to make this refactor but there seems to be an issue using `import * as git from '../../../util/git';` that I have been unable to resolve. 
When the git import is introduced `yarn test` throws ~270 errors when running `tsc --noEmit -p tsconfig.strict.json` which is beyond my level of understanding

Reply:
You need to add the utils file to the exclude list in `tsconfig.strict.json`

Reply:
Amazing, thank you for pointing that out! I was not going to solve that myself. Moved `getRepoGitUrl` into utils, let me know if there are any issues 😄

Reply:
https://github.com/renovatebot/renovate/blob/main/.github/contributing.md#re-requesting-a-review 😉